configs/body_2d_keypoint/topdown_heatmap/coco/td-hm_hrnet-w32_udp-regress-8xb64-210e_coco-256x192.py

```diff
@@ -83,7 +83,8 @@
         in_channels=32,
         out_channels=3 * 17,
         deconv_out_channels=None,
-        loss=dict(type='KeypointMSELoss', use_target_weight=True),
+        loss=dict(type='CombinedTargetMSELoss', use_target_weight=True),
+        print_acc_pose=False,
```

Comment:
```suggestion
        compute_acc=False,
```

Reply:
Another option is to get it from `train_cfg`, like:
```
compute_acc = train_cfg.get('compute_acc', True)
```

Reply:
Agreed. I think put it into `train_cfg` will be more reasonable.